Program.cs

```diff
@@ -152,6 +154,26 @@ [12] All
             
         }
 
+        public static string[] ParseGlobalFlags(string[] args)
+        {
+            List<string> arguments = new List<string>(args);
+            var custom_user_data_flag = arguments.IndexOf("/custom_browser");
+            // If flag is not shown - ignore
+            if (custom_user_data_flag == -1)
+                return args;
+            else if (args.Length >= custom_user_data_flag + 1)
```

Comment:
no need for the `else`

Reply:
Actually, it's needed since there could be a case where the `/custom_browser` exists but without a follow-up parameter:
`args.Length == custom_user_data_flag`